src/main/features/claim/views/eligibility/not-eligible.njk

```diff
@@ -60,6 +60,11 @@
         <p>{{ t('You might be able to get advice from organisations like ' +
           '<a target="_blank" href="{{ citizensAdviceUrl }}">Citizens Advice</a> ' + newWindow() + ' about making a claim.',
           { citizensAdviceUrl: citizensAdviceUrl }) | safe }}</p>
+      {% elseif reason === NotEligibleReason.UNDER_18_DEFENDANT %}
+        <p>{{ t('You can only use this service to claim against a defendant who’s 18 or over.') }}</p>
+        <p>{{ t('You might be able to get advice from organisations like ' +
+            '<a target="_blank" href="{{ citizensAdviceUrl }}">Citizens Advice</a> ' + newWindow() + ' about making a claim.',
```

Comment:
Please use `features/response/views/eligibility/send-your-response-by-email.njk:8` macro here?

Reply:
I don't think you linked to the correct file? what has response by email to do with citizens advice?

Reply:
I linked the right file, I meant to start using that new macro application wise. It was made local when response was built, now it is time to move it to common space (not common-frontend) and start reusing.